src/dsp/EQ/EQProcessor.cpp

```diff
@@ -37,6 +41,11 @@ void EQProcessor::processBlock (const chowdsp::BufferView<float>& buffer)
 
         chowdsp::BufferMath::copyBufferData (doubleBuffer, buffer);
     }
+
+    juce::AudioBuffer<float> audioBuffer;
+    audioBuffer.setSize (numChannels, numSamples);
+    chowdsp::BufferMath::copyBufferData (buffer, audioBuffer);
+    eqHelper->processBlockInput (audioBuffer);
```

Comment:
I think we can simplify this to:
```cpp
eqHelper->processBlockInput (buffer.toAudioBuffer());
```

Although we might need to adjust `processBlockInput()` to take a const buffer.